### PR TITLE
Scavenger drone,Admin sus drone, flightdeck clothing, CEC and SCAF posters, large SCAF floor decals

### PR DESCRIPTION
**Pull request by blackwolf602**
Adds the scavenger bot from DS3 in a player controlled drone form, craft able, hold able, and wearable. adds a admin spawnable sus drone as a joke. adds CEC and SCAF posters, large SCAF floor decals, flightdeck clothing

### CODE REVIEW

File: necromorphs/code/modules/mob/living/inhand_holder.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/obj/item/mob_holder/scavbot
+	examine_mob = FALSE
+
+/obj/item/mob_holder/drone/scavbot/deposit(mob/living/target_mob)
```
Comment (DTraitor) [inline necromorphs/code/modules/mob/living/inhand_holder.dm:4]: Are the types supposed to be different?

File: necromorphs/code/modules/mob/living/simple_animal/friendly/drone/Scavengerbot.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	health = 120
+	maxHealth = 120
+	sight = (SEE_OBJS)
+	held_type = /obj/item/mob_holder/scavbot
+	worn_slot_flags = (ITEM_SLOT_BACK | ITEM_SLOT_BELT)
+	initial_language_holder = /datum/language_holder/synthetic
+	laws = \
+	"1. Collect small out of the way resoruces and items for use as resources.\n"+\
+	"2. Deliver the resources and found items to the appropriate department.\n"+\
+	"3. You may not harm any being, regardless of intent or circumstance"
```
Comment (DTraitor) [inline necromorphs/code/modules/mob/living/simple_animal/friendly/drone/Scavengerbot.dm:21]: Shouldn't third law be the first one?

File: necromorphs/code/modules/mob/living/simple_animal/friendly/drone/construction.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+					qdel(src)
+
+/obj/item/wallframe/apc/attackby(obj/O, mob/user, params)
+	if(istype(O, /obj/item/bodypart/arm/left/robot) || istype(O, /obj/item/bodypart/arm/right/robot))
+		to_chat(user, span_notice("You add [O] to [src]."))
+		qdel(O)
+		qdel(src)
+		user.put_in_hands(new /obj/item/bot_assembly/scavbot)
+	else
+		..()
```
Comment (DTraitor) [inline necromorphs/code/modules/mob/living/simple_animal/friendly/drone/construction.dm:49]: Put this code in the APC code

File: necromorphs/code/modules/mob/living/inhand_holder.dm (hunk 1 above)
Comment (blackwolf602) [reply necromorphs/code/modules/mob/living/inhand_holder.dm:4]: nah

File: necromorphs/code/modules/mob/living/simple_animal/friendly/drone/Scavengerbot.dm (hunk 2 above)
Comment (blackwolf602) [reply necromorphs/code/modules/mob/living/simple_animal/friendly/drone/Scavengerbot.dm:21]: good idea, fixed

File: necromorphs/code/modules/mob/living/simple_animal/friendly/drone/construction.dm (hunk 3 above)
Comment (blackwolf602) [reply necromorphs/code/modules/mob/living/simple_animal/friendly/drone/construction.dm:49]: done

**Comment by blackwolf602** [comment]
I removed rigs_clothing since we are storing rig sprites in the mod suit area

**Pull request closed without merging**